Include deployment modes in responses if defined at inputs of the policy templates (#1345)
elastic/package-registry: CHANGELOG.md

* Add new method `MustParsePackage` to create new packages, running more validations. [#1333](https://github.com/elastic/package-registry/pull/1333)
* Allow to customize settings related to SQL storage indexer (technical preview). [#1334](https://github.com/elastic/package-registry/pull/1334) [#1337](https://github.coim/elastic/package-registry/pull/1337)
* Cleanup SQL storage indexer backup database only when a new index version is downloaded (technical preview). [#1337](https://github.coim/elastic/package-registry/pull/1337)

> * Allow to define deployment modes in each input of the policy templates. [#1345](https://github.coim/elastic/package-registry/pull/1345)

Reviewer: Nit. The change that allows defining deployment modes is in the Spec, maybe here we should say that deployment modes are included in responses.
```suggestion
* Include deployment modes in responses when they are defined at inputs of the policy templates. [#1345](https://github.coim/elastic/package-registry/pull/1345)
```
